erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py

```diff
@@ -209,7 +209,7 @@ def new_bank_transaction(transaction):
 			result.append(new_transaction.name)
 
 		except Exception:
-			frappe.throw(frappe.get_traceback())
+			frappe.throw(title=_('Bank transaction creation error'))
```

Comment:
This is just title, what about error message?

Reply:
If the message is not specified, traceback becomes the error message. https://github.com/frappe/frappe/blob/90b4af0662d5b125108514b7d78195189abcd083/frappe/__init__.py#L1567